lib/hammer_cli/settings.rb

```diff
@@ -40,8 +40,8 @@ def self.load_from_file(file_path)
             load(config)
             path_history << file_path
           end
-        rescue Exception
-          puts _("Couldn't load configuration file %s, please check the YAML file for syntax.") % file_path
+        rescue Exception => e
+          puts _("Couldn't load configuration file %{path}: %{message}") % { path: file_path, message: e.message }
```

Comment:
After some testing I'd prefer to use warning `warn _("Warning: Couldn't......`.
It feels better visible and more in line with other similar output we do. What do you think?